src/compounds/Item/Item.jsx

```diff
@@ -115,6 +115,11 @@ Item.defaultProps = {
   isLoading: false,
   item: {
     description: '',
+    id: null,
+    img: {
+      alt: '',
+    },
```

Comment:
I see that it isn't, but the entire `img` prop should be required for this component. and since `img` is also required, we don't want to define any of it.

Reply:
```suggestion
```